po/fr.po

```diff
@@ -44466,6 +44517,129 @@ msgstr "Ouvrir le profil spécifié dans un éditeur"
 msgid "Also print directory history"
 msgstr ""
 
+#: /tmp/fish/implicit/share/completions/ngrok.fish:1
+#, fuzzy
+msgid "Save authtoken to configuration file"
+msgstr "Paramétrer l’emplacement du fichier de configuration alternatif"
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:2
+#, fuzzy
+msgid "Prints author and licensing information"
+msgstr "Afficher les droits d’auteur et les informations de version"
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:3
+msgid "Start an HTTP tunnel"
+msgstr ""
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:4
+#, fuzzy
+msgid "Start tunnels by name from the configuration file"
+msgstr "Paramétrer l’emplacement du fichier de configuration alternatif"
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:5
+msgid "Start a TCP tunnel"
+msgstr ""
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:6
+msgid "Start a TLS tunnel"
+msgstr ""
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:7
+#, fuzzy
+msgid "Update ngrok to the latest version"
+msgstr "Mettre à jour vers la dernière version"
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:8
+#, fuzzy
+msgid "Print the version string"
+msgstr "Afficher la version"
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:9
+#, fuzzy
+msgid "Shows a list of commands or help for one command"
+msgstr "Afficher l’aide pour une commande"
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:10
+msgid "ngrok.com authtoken identifying a user"
+msgstr ""
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:11
+msgid "path to config files; they are merged if multiple"
+msgstr ""
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:12
+msgid "path to log file, 'stdout', 'stderr' or 'false'"
+msgstr ""
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:13
+msgid "log record format: 'term', 'logfmt', 'json'"
+msgstr ""
+
+#: /tmp/fish/implicit/share/completions/ngrok.fish:14
+#, fuzzy
+msgid "logging level"
+msgstr "Niveau de compression"
```

Comment:
incorrect

Reply:
Welcome to my world. `make po/fr.po` tries to fuzzily complete empty translations by searching a translated close one, which makes stupid translations like that. Nevertheless, note they are fuzzy, i.e. marked as unreliable.

Reply:
I see. I'm fixing some of them now, particularly those that appear in my diffs, but not everything. And http://bitoduc.fr has been of no help today.

Your French is flawless btw